package.json

```diff
@@ -40,16 +40,16 @@
     "node": ">=8.0.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.8.4",
-    "@babel/core": "^7.9.6",
-    "@babel/preset-env": "^7.9.6",
-    "@metarhia/doc": "^0.6.0",
+    "@babel/cli": "^7.10.1",
+    "@babel/core": "^7.10.2",
+    "@babel/preset-env": "^7.10.2",
+    "@metarhia/doc": "^0.6.1",
     "eslint": "^7.1.0",
-    "eslint-config-metarhia": "^7.0.0",
+    "eslint-config-metarhia": "^7.0.1",
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-import": "^2.20.2",
     "eslint-plugin-prettier": "^3.1.3",
     "metatests": "^0.7.1",
-    "prettier": "1.18.x"
+    "prettier": "^1.18.0"
```

Comment:
I'd prefer to update Prettier in a separate PR that would also introduce the resulting changes to the code after releasing a new major version of this package without Node.js 8 support.